ironfish-cli/src/ui/confirm.ts

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is unchanged. It was moved out of the `ui.ts` file, which was broken into smaller pieces